File: BackOfficeAPI/Core/Entities/Brand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.Entities
{
    public class Brand: BaseEntity
    {
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? ShortDescription { get; set; }
        public string? BrandImage { get; set; }
        public Branch Branch { get; set; }
        public int BranchID { get; set; }
        public bool Active { get; set; }

    }
}

File: Ecart/src/app/shared/models/brand.ts
import { commonParams } from "./commonparams";

export interface IBrand {
    pageIndex: number;
    pageSize: number;
    count: number;
    records: IBrandList[];
}

export interface IBrandList {
    id: number;
    name: string;
    description: string;
    shortDescription: string;
    brandImage: string;
    branchID: number;
    branch: string;
}

export class brandParams extends commonParams {
    branchID?:number;
}

